opentelemetry-extra/exe/eventlog-to-tracy/Main.hs

```diff
@@ -1,15 +1,37 @@
 module Main where
 
+import OpenTelemetry.ChromeExporter
+import OpenTelemetry.EventlogStreaming_Internal
+import OpenTelemetry.Exporter
+import System.Clock
 import System.Environment (getArgs)
 import System.Exit
 import System.Process
+import System.Directory
 
 help :: IO ()
 help = do
   putStrLn "Converts eventlog to Tracy format and launches the viewer."
   putStrLn "Path to eventlog is expected."
   exitFailure
 
+callProcessOrExplain :: FilePath -> String -> [String] -> IO ()
+callProcessOrExplain filename instruction args = do
+  exists <- findExecutable filename
+  case exists of
+    Just _ -> callProcess filename args
+    Nothing -> do
+      putStrLn $ filename <> ": command not found, please install " <> instruction
+      exitFailure
+
+eventlogToChrome :: FilePath -> FilePath -> IO ()
```

Comment:
This function should probably live in `OpenTelemetry.ChromeExporter` since it's not Tracy-specific.

Reply:
Everything else looks good to me.